main.go

```diff
@@ -22,7 +22,13 @@ type Params struct {
 	Username      *string  `json:"username,omitempty"`
 }
 
+var (
+	buildDate string
```

Comment:
I like repeatable builds, it's nice to know that you're running exactly what your "customer" was running when trying to debug an issue. That said, it's probably not critical here, versus say some firmware image for an embedded device with millions of lines of code.

This change isn't a showstopper for me. (I do like your idea of a git sha rather than a date)